docs/contributor/02-10-operations.md

```diff
@@ -26,77 +26,77 @@ the prerequisites yourself.
 
 ![Provisioning diagram](/docs/assets/provisioning.svg)
 
-The provisioning process is part of a module reconciliation. To trigger the reconciliation, create a [BtpOperator CR](/api/v1alpha1/btpoperator_types.go):
-
-```shell
-cat <<EOF | kubectl apply -f -
-apiVersion: operator.kyma-project.io/v1alpha1
-kind: BtpOperator
-metadata:
-  name: btpoperator
-  labels:
-    app.kubernetes.io/managed-by: btp-manager
-EOF
-```
-
-The BtpOperator reconciler picks up the created CR and determines whether it should be responsible for representing the
-module status. The BtpOperator CR reflects the status of the operand, that is, SAP BTP Service Operator, only when it is
-the oldest CR present in the cluster. In that case a finalizer is added, the CR is set to `Processing` state and the
+The provisioning process is part of a module reconciliation. 
+1. To trigger the reconciliation, create a [BtpOperator CR](/api/v1alpha1/btpoperator_types.go):
+
+   ```shell
+   cat <<EOF | kubectl apply -f -
+   apiVersion: operator.kyma-project.io/v1alpha1
+   kind: BtpOperator
+   metadata:
+     name: btpoperator
+   EOF
+   ```
+
+2. The BtpOperator reconciler picks up the created CR and determines whether it should be responsible for representing the
+module status. 
+3. The BtpOperator CR reflects the status of the operand, that is, SAP BTP Service Operator, only when it is
+the oldest CR present in the cluster. Otherwise, it is given the `Error` state (3a) with the condition reason `OlderCRExists` and the message containing details about the CR responsible for reconciling the operand.
+4. For the only or the oldest CR present in the cluster,  a finalizer is added, the CR is set to the `Processing` state, and the
 reconciliation proceeds.
-Otherwise, it is given the `Error` state with the condition reason `OlderCRExists` and message containing details
-about the CR responsible for reconciling the operand.
+5. The reconciler looks for a `sap-btp-manager` Secret in the `kyma-system` Namespace with the label `app.kubernetes.io/managed-by: kcp-kyma-environment-broker`. This Secret contains the Service
```

Comment:
Please join all parts of the sentence in one line (47+49+50..) to fix the formatting of the ordered list.